contributors/devel/development.md

```diff
@@ -101,13 +113,25 @@ revising your hypothesis.  The above issues do this transparently, using figures
 and data rather then conjecture. Notice that the problem is analyzed and a
 correct solution is created before a single line of code is reviewed.
 
-## Building Kubernetes with Docker
+## Getting the code
+
+Kubernetes is developed using git for version control.
+
+### Install
+
+Follow the [official git documentation] to learn how to install git, then follow the directions in [github-workflow.md] for step-by-step development procedures.
+
+## Building Kubernetes
+
+There are two recommended ways of building Kubernetes, and the choice of one or the other depends on if you do plan on developing upstream. One way is building a release from source, this is recommended for development purposes. If there is no development involved, it is suggested using a pre-built version of the current release.
+
+### Building Kubernetes with Docker
 
 Official releases are built using Docker containers. To build Kubernetes using
 Docker please follow [these
 instructions](http://releases.k8s.io/HEAD/build/README.md).
 
-## Building Kubernetes on a local OS/shell environment
+### Building Kubernetes on a local OS/shell environment
 
 Kubernetes development helper scripts assume an up-to-date GNU tools
 environment. Recent Linux distros should work out-of-the-box.
```

Comment:
s/distros/distributions/

Reply:
Should I replace 'local OS/shell' for distributions, right?

Reply:
No, but "Recent Linux distros" -> "Recent Linux distributions".

Reply:
@eduartua please, keep "OS/shell environment" as before :)

My suggestion was about line 137. Now: "Recent Linux distros should work out-of-the-box.", should be "Recent Linux distributions should work out-of-the-box."

Reply:
Thank you!